test/script/reference.t

```diff
@@ -1,7 +1,11 @@
   $ owi script --no-exhaustion reference/address.wast
   $ owi script --no-exhaustion reference/align.wast
   $ owi script --no-exhaustion reference/binary-leb128.wast
+  expected integer representation too long but there was no error
+  [7]
```

Comment:
This should be fixed in the binary parser.

Reply:
This is because we ignore custom sections instead of validating them before ignoring them. I gave a minimal example to reproduce as a comment in `binary_parser.ml`.